fullstack-cert/typescript-projects/flashcard-quiz-app/src/script.ts

```diff
@@ -0,0 +1,226 @@
+const cardDisplay = document.querySelector("#current-card") as HTMLElement;
+const cardButtonsContainer = document.getElementById(
+  "cards-list",
+) as HTMLElement;
+
+const frontInput = document.getElementById("front-text") as HTMLTextAreaElement;
+const backInput = document.getElementById("back-text") as HTMLTextAreaElement;
```

Comment:
```suggestion
const cardDisplay = document.querySelector<HTMLElement>("#current-card");
const cardButtonsContainer = document.getElementById<HTMLElement>(
  "cards-list",
);

const frontInput = document.getElementById<HTMLTextAreaElement>("front-text");
const backInput = document.getElementById<HTMLTextAreaElement>("back-text");
```

Rather than type casting, we should teach campers to pass the generic.